src/chat/dto/post-chat.dto.ts

```diff
@@ -1,20 +1,50 @@
 import { ApiProperty } from '@nestjs/swagger';
-import { IsNotEmpty, IsNumber, IsString } from 'class-validator';
+import {
+  IsMongoId,
+  IsNotEmpty,
+  IsNumber,
+  IsOptional,
+  IsString,
+} from 'class-validator';
+import mongoose from 'mongoose';
 
 export class PostChatDto {
+  @IsMongoId()
+  @IsNotEmpty()
+  @ApiProperty({
+    example: '650bde3798dd4c34439c30dc',
+    description: '채팅을 전송하는 채팅방 id',
+  })
+  roomId: mongoose.Types.ObjectId;
+
   @ApiProperty({
     example: '안녕하세요',
     description: '채팅 내용',
   })
   @IsString()
-  @IsNotEmpty()
+  @IsOptional()
```

Comment:
```suggestion
    @ApiPropertyOptional({
    example: '안녕하세요',
    description: '채팅 내용',
  })
  @IsString()
  @IsNotEmpty()
  @IsOptional()
  content?: string;
```
옵셔널 프로퍼티라면 다음과같이 수정하는것이 좋을듯해요.
1. ApiPropertyOptional 데코레이터를 사용함으로써 swagger 상에 optional 프로퍼티임을 명시합니다. ApiProperty 데코레이터의 required 필드를 false 로 설정하는 방법도 있습니다.
2. optional 프로퍼티라는것은 들어올 수도있고 들어오지 않을수도있기때문에 typescript 의 `?` 를 사용하여 타입스크립트의 강점을 더 활용할 수 있습니다.